README.md

```diff
@@ -15,10 +15,12 @@
   ~ limitations under the License.
   -->
 
-# What is Uniffle
+# Apache Uniffle (Incubating)
 
-Uniffle is a Remote Shuffle Service, and provides the capability for Apache Spark applications
-to store shuffle data on remote servers.
+Uniffle is a unified remote shuffle service for analytics engines.
+It provides the ability to aggregate and store shuffle data on remote servers,
```

Comment:
aggregate?

Reply:
I mean "batching writes". Any better word for this?

Reply:
buffer?